libsweep/README.md

```diff
@@ -245,7 +289,16 @@ Opaque type representing a single full 360 degree scan from a `sweep_device_s`.
 sweep_scan_s sweep_device_get_scan(sweep_device_s device, sweep_error_s* error)
 ```
 
-Blocks waiting for the `sweep_device_s` to accumulate a full 360 degree scan into `sweep_scan_s`.
+Returns the ordered readings (1st to last) from a single scan.
+Retrieves the oldest scan from a queue of scans accumulated in a background thread. Blocks until a scan is available. To be used after calling `sweep_device_start_scanning`, NOT after `sweep_device_attempt_start_scanning`.
```

Comment:
For non low-level development, the API should be unchanged. The "attempt_..." methods are provided as niche case alternatives... and not meant to be advertised. I went ahead and separated these methods out of the main README so that the distinction is clear. See if that makes more sense. 

If you have ideas for a better naming convention, please share! I'm not very content with the current "attempt_" and "direct_" pattern.